src/components/volunteer/VolunteerGroupSearchField.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { colors, Icon, normalize, texts } from '../../config';
+import { Input } from '../form';
+import { WrapperRow, WrapperVertical } from '../Wrapper';
+
+export const VolunteerGroupSearchField = ({ onPress }: { onPress: () => void }) => {
+  const { control } = useForm({
+    defaultValues: {
+      dummy: ''
+    }
+  });
+
+  return (
+    <WrapperVertical>
+      <TouchableOpacity onPress={onPress}>
+        <WrapperRow pointerEvents="none">
+          <Input
+            disabled
+            chat
+            placeholder={texts.volunteer.groupSearchNew}
+            name="dummy"
+            control={control}
```

Comment:
Why did you choose the name dummy here? I think this is search input. can it be called `search`?